akka-remote/src/main/resources/reference.conf

```diff
@@ -641,17 +641,17 @@ akka {
         # still set to 'off' per default.
         #
         # Note: Nodes that are configured with this setting to 'on' might not be able to receive messages from nodes that
-        # run on older versions of akka-remote. This is because in older versions of Akka the active side of the remoting
-        # connection will not send over certificates.
+        # run on older versions of akka-remote. This is because in versions of Akka < 2.4.12 the active side of the remoting
+        # connection will not send over certificates even if asked.
         #
-        # However, starting from the version this setting was added, even with this setting "off", the active side
-        # (TLS client side) will use the given key-store to send over a certificate if asked. A rolling upgrades from
-        # older versions of Akka can therefore work like this:
-        #   - upgrade all nodes to an Akka version supporting this flag, keeping it off
-        #   - then switch the flag on and do again a rolling upgrade of all nodes
+        # However, starting with Akka 2.4.12, even with this setting "off", the active side (TLS client side)
+        # will use the given key-store to send over a certificate if asked. A rolling upgrade from versions of
+        # Akka < 2.4.12 can therefore work like this:
+        #   - upgrade all nodes to an Akka version >= 2.4.12, in the best case the latest version, but keep this setting at "off"
+        #   - then switch this flag to "on" and do again a rolling upgrade of all nodes
```

Comment:
Yes, I guess makes sense if people update from an older Akka version. Will add that.